<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroFuncionarios</class>
 <widget class="QDialog" name="CadastroFuncionarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro Funcionário</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:rgb(255,255,255);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>511</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCadastroFuncionario_CPF">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCadastroFuncionario_Nome">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Salário</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCadastroFuncionario_Salario">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Departamento</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbCadastroFuncionario_Departamento">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Esolha o departamento desejado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Administrativo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comercial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Financeiro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jurídico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Marketing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Operacional</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Recursos Humanos</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data de Nascimento</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCadastroFuncionario_DataDeNascimento">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telefone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCadastroFuncionario_Telefone">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCadastroFuncionario_Email">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>511</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSalvarFuncionario">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Salvar</string>
      </property>
      <property name="icon">
       <iconset resource="recursosimagens.qrc">
        <normaloff>:/Imagens/Imagens/salvar.png</normaloff>:/Imagens/Imagens/salvar.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelarFuncionario">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="recursosimagens.qrc">
        <normaloff>:/Imagens/Imagens/cancelar.png</normaloff>:/Imagens/Imagens/cancelar.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursosimagens.qrc"/>
 </resources>
 <connections/>
</ui>
